activerecord/lib/active_record/railties/databases.rake

```diff
@@ -152,6 +176,25 @@ db_namespace = namespace :db do
       db_namespace["_dump"].invoke
     end
 
+    namespace :down do
+      ActiveRecord::Tasks::DatabaseTasks.for_each do |spec_name|
+        task spec_name => :load_config do
```

Comment:
@eileencodes : `desc` is missing for `down` here. Just like you have added here(https://github.com/rails/rails/pull/36039/files#diff-28a5ae383b291583c513ad8eeed99a3aR146). Do you think we should add it?

Reply:
Oops I left that in accidentally. Fixed in 5cc7466031d7550db44441361760c6e3c34f0291